### PR TITLE
Adds warnings for split packages in maven build

### PR DESCRIPTION
**Pull request by reinhapa**

### CODE REVIEW

File: src/main/java/org/adoptopenjdk/maven/plugins/OutputParser.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    public void parse(byte[] outputData) throws IOException {
+        Pattern pattern = Pattern.compile("^[\\s]+([0-9]+)[\\s]+(.*)$");
+        try (BufferedReader reader = new BufferedReader(new InputStreamReader(new ByteArrayInputStream(outputData)))) {
+            Set<ModuleDetail> details = new HashSet<>();
+            String currentPackage = null;
+            String line = reader.readLine();
+            while ((line = reader.readLine()) != null) {
+                Matcher matcher = pattern.matcher(line);
+                if (matcher.matches()) {
+//                    System.out.println("matches " + line + "\t" + matcher.group(1) + "\t" + matcher.group(2));
```
Comment (karianna) [inline src/main/java/org/adoptopenjdk/maven/plugins/OutputParser.java]: Remove commented out code

File: src/test/java/org/adoptopenjdk/maven/plugins/JsplitpkgscanMojoTest.java [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+public class JsplitpkgscanMojoTest extends AbstractMojoTestCase {
+    private Tool tool;
+    private JsplitpkgscanMojo jsplitpkgscanMojo;
+
+    /**
+     * {@inheritDoc}
+     */
+    protected void setUp()
+            throws Exception {
+        // required
```
Comment (karianna) [inline src/test/java/org/adoptopenjdk/maven/plugins/JsplitpkgscanMojoTest.java]: What is required and why?

File: src/test/java/org/adoptopenjdk/maven/plugins/JsplitpkgscanMojoTest.java [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+    }
+
+    static class OutputStreamMatcher implements ArgumentMatcher<OutputStream> {
+        @Override
+        public boolean matches(OutputStream outputStream) {
+            try {
+                File outFile = getTestFile("src/test/resources/jsplitpkgscan.out");
+                Files.copy(outFile.toPath(), outputStream);
+            } catch (IOException e) {
+                e.printStackTrace();
```
Comment (karianna) [inline src/test/java/org/adoptopenjdk/maven/plugins/JsplitpkgscanMojoTest.java]: send to logger instead

File: src/test/java/org/adoptopenjdk/maven/plugins/OutputParserTest.java [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+import static org.easymock.EasyMock.*;
+import static org.mockito.Mockito.only;
+
+public class OutputParserTest {
+    private byte[] outputData;
+
+    @Before
+    public void setUp() throws IOException {
+        try (InputStream in = getClass().getResourceAsStream("/jsplitpkgscan.out");
+             ByteArrayOutputStream out = new ByteArrayOutputStream()) {
```
Comment (karianna) [inline src/test/java/org/adoptopenjdk/maven/plugins/OutputParserTest.java]: whitespace

File: src/main/java/org/adoptopenjdk/maven/plugins/OutputParser.java (hunk 1 above)
Comment (reinhapa) [reply src/main/java/org/adoptopenjdk/maven/plugins/OutputParser.java]: Removed all comments

File: src/test/java/org/adoptopenjdk/maven/plugins/JsplitpkgscanMojoTest.java (hunk 2 above)
Comment (reinhapa) [reply src/test/java/org/adoptopenjdk/maven/plugins/JsplitpkgscanMojoTest.java]: Added an override annotation and removed the comment-

File: src/test/java/org/adoptopenjdk/maven/plugins/JsplitpkgscanMojoTest.java (hunk 3 above)
Comment (reinhapa) [reply src/test/java/org/adoptopenjdk/maven/plugins/JsplitpkgscanMojoTest.java]: Re-throwed the exception as assertion error

File: src/test/java/org/adoptopenjdk/maven/plugins/OutputParserTest.java (hunk 4 above)
Comment (reinhapa) [reply src/test/java/org/adoptopenjdk/maven/plugins/OutputParserTest.java]: Revormatted

**Pull request merged**